k8s/demo/galera-xtradb-cluster/deployments/README.md

```diff
@@ -0,0 +1,265 @@
+# Running Percona Galera Cluster with OpenEBS 
+
+This tutorial provides detailed instructions to perform the following tasks :
+
+- Run a 3-node percona galera cluster with OpenEBS storage in a Kubernetes environment
+- Test the data replication across the percona xtradb mysql instances
+
+## Galera Cluster
+
+Percona XtraDB Cluster is an active/active high availability and high scalability open source solution for MySQL clustering. 
```

Comment:
active/active??

Reply:
its is a term widely used to describe a type  of HA mode